requirements/base.txt

```diff
@@ -34,7 +34,7 @@ PyJWT==2.4.0
 pyparsing==3.0.9
 python-dateutil==2.8.2
 python-dotenv==0.20.0
-python-magic==0.4.27
+python-magic==0.4.15
```

Comment:
Is it intended? All is working with this version?

Reply:
Revert changes and reinstall locally version 0.4.27 because it has been tested (proven) and it already works for us.

Reply:
it didn't work for me with the latest version, I don't know why. Reverted dependencies to 4.27